src/EnergyPlus/HVACControllers.cc

```diff
@@ -179,8 +179,10 @@ using namespace DataHVACControllers;
 using namespace DataRootFinder;
 
 // Number of significant digits to display in error messages for floating-point numbers
-constexpr Real64 SomeFloatingPoint(1.0);
-int const NumSigDigits(PRECISION(SomeFloatingPoint));
```

Comment:
This was way too much effort put into determining a number of significant digits for reporting.  I evaluated it and found it to be 15.  Removing this one usage of PRECISION allowed me to clean out a significant chunk of Objexx code.

Reply:
We're reporting 15 significant digits? Is anything in a building known to even 5 significant digits?